recipes/learn-python-ceg-test/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "learn-python-ceg-test" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/learn_python_ceg_test-{{ version }}.tar.gz
+  sha256: afb3dfce422d1315869b066101d807ab48d4cfdb099e415df5d246482459b0c5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pandas
+
+test:
+  imports:
+    - learn_python_ceg_test
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mike-mendoza/learn_python_ceg_test
+  summary: learn_python check answers
+  license: GPL-3.0

```

This should satisfy the linter.

```suggestion
  license: GPL-3.0-Only
```